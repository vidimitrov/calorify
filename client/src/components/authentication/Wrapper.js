import styled from 'styled-components';

const Wrapper = styled.div`
  display: flex;
  align-items: center;
  flex-direction: column; 
  justify-content: center;
  width: 100%;
  min-height: 100%;
  padding: 20px;
  background-color: #f0cd61;
  height: 100vh;
  box-sizing: border-box;
`;

export default Wrapper;
