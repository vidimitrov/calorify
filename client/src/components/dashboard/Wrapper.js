import styled from 'styled-components';

const Wrapper = styled.div`
  background-color: #ffffff;
  height: 100vh;
`;

export default Wrapper;
