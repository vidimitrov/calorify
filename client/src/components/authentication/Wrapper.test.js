import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Wrapper from './Wrapper';

it('Authentication/Wrapper component renders correctly', () => {
  const tree = renderer.create(<Wrapper><div /></Wrapper>).toJSON();
  expect(tree).toMatchSnapshot();
});
