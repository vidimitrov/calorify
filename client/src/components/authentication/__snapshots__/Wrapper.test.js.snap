// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/Wrapper component renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  min-height: 100%;
  padding: 20px;
  background-color: #f0cd61;
  height: 100vh;
  box-sizing: border-box;
}

<div
  className="c0"
>
  <div />
</div>
`;
